egs/wsj/asr1/run.sh

```diff
@@ -25,6 +26,8 @@ elayers=6
 eunits=320
 eprojs=320
 subsample=1_2_2_1_1 # skip every n frame from input to nth layers
+# loss related
+ctctype=chainer
```

Comment:
@sw005320 do you think that `warpctc` should be default?

Reply:
I'll decide after I tested it with the other recipes. For now, it's OK to make chainer one as default.